setup/bundles.rst

```diff
@@ -173,7 +173,7 @@ Before diving into the specifics of the most common edge cases, the general
 recommendation is to **not rely on the Symfony Kernel version** to decide which
 code to use::
 
-    if (Kernel::VERSION_ID <= 20800) {
+    if (Kernel::VERSION_ID < 20800) {
```

Comment:
Most of the time, you do want to get the shift on `2.8.0`, that's what we do on sonata : 
```php
if ($conditionAbout2.8) {
    // use  new way, that way we don't get the deprecation notice
} else {
   // use old way
}
```

See for instance : https://github.com/sonata-project/SonataAdminBundle/blob/48cc780dbcd026193e0e98fb2ad6a2a997e2ec22/Form/Type/AclMatrixType.php#L39